src/neo-vm/ExecutionEngine.cs

```diff
@@ -184,42 +173,132 @@ private bool ExecuteInstruction()
                 // Control
                 case OpCode.NOP: break;
                 case OpCode.JMP:
+                    {
+                        return ExecuteJump(true, instruction.TokenI8);
```

Comment:
What do you think if we have two methods, one for `true`, and other for `false`? In order to prevent to send the parameter and check this value when we already knows the result

Reply:
Since `ExecuteJump` is inlined, the overhead is very little, and may be optimized by compiler.